src/test/TxTests.cpp

```diff
@@ -66,6 +66,17 @@ ExpectedOpResult::ExpectedOpResult(AccountMergeResultCode accountMergeCode)
     mOperationResult.tr().accountMergeResult().code(accountMergeCode);
 }
 
+ExpectedOpResult::ExpectedOpResult(AccountMergeResultCode accountMergeCode,
+                                   int64_t sourceAccountBalance)
+{
+    assert(accountMergeCode == ACCOUNT_MERGE_SUCCESS);
```

Comment:
why not throw instead of assert? I think `assert` messes up the way failures are reported by the catch library.